spring-boot-postgresql/README.md

```diff
@@ -0,0 +1,44 @@
+# Spring Boot PostgreSQL Fragment
+
+## Using the fragment
+
+To include this fragment you should add an import to the `accelerator` section:
+
+```yaml
+accelerator:
+  # ...
+  imports:
+  - name: spring-boot-postgresql
+    
+```
+
+Then in your `engine` section add an `InvokeFragment` directive at an appropriate level.
+
+```yaml
+engine:
+  # ...
+    - type: InvokeFragment
+      reference: spring-boot-postgresql
+      let:
+        - name: todododo
```

Comment:
I assume this was written early and you forgot to revisit?

Reply:
Yup.... fixed. Thanks.